testing/fake_certs.go

```diff
@@ -319,16 +319,12 @@ func DefaultAsvk() (*rsa.PrivateKey, error) {
 
 // DefaultVcek returns a new ECDSA key on the expected curve for a VCEK.
 func DefaultVcek() (*ecdsa.PrivateKey, error) {
-	privateKey, err := ecdsa.GenerateKey(elliptic.P384(), insecureRandomness)
```

Comment:
Would you also create pre-generated default ask, asvk, and ark private keys? That would make the test suites run a lot faster I think.

Reply:
Done.